src/Pmi/Application/AbstractApplication.php

```diff
@@ -277,14 +275,28 @@ public function setConfig($key, $val)
     public function getGoogleServiceClass()
     {
         return $this['isUnitTest'] ? 'Tests\Pmi\GoogleUserService' :
-            'google\appengine\api\users\UserService';
+            'Pmi\Service\MockGoogleUserService';
+    }
+
+    /** For local development. */
+    public function setMockUser($email)
+    {
+        $cls = $this->getGoogleServiceClass();
+        if (class_exists($cls)) {
+            $cls::setMockUser($email);
+            $this['session']->set('mockUser', $cls::getCurrentUser());
+        }
     }
 
     public function getGoogleUser()
     {
-        if ($this->getConfig('gae_auth')) {
-            $cls = $this->getGoogleServiceClass();
-            return class_exists($cls) ? $cls::getCurrentUser() : null;
+        if ($this->isLocal() && $this->getConfig('gae_auth')) {
```

Comment:
Can we change this config name from `gae_auth` to something like `local_mock_auth`, since we are no longer using the old GAE auth workflow.